docs/quickstarts/ratify-with-venafi.md

```diff
@@ -0,0 +1,209 @@
+    --name-template=gatekeeper \
+    --namespace gatekeeper-system --create-namespace \
+    --set enableExternalData=true \
+    --set validatingWebhookTimeoutSeconds=5 \
+    --set mutatingWebhookTimeoutSeconds=2
+```
+
+Next, we need to deploy a Gatekeeper policy and constraint. For this guide, we will use a sample policy and constraint that requires images to have at least one trusted signature.
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/deislabs/ratify/main/library/notation-validation/template.yml
+kubectl apply -f https://raw.githubusercontent.com/deislabs/ratify/main/library/notation-validation/samples/constraint.yaml
+```
+
+More complex combinations of regos and Ratify verifiers can be used to accomplish many types of checks. See the [Gatekeeper docs](https://open-policy-agent.github.io/gatekeeper/website/docs/) for more information on rego authoring.
+
+## Deploy Ratify
+
+Now we can deploy Ratify to our cluster:
+```shell
+helm install ratify \
+    ratify/ratify --atomic \
+    --namespace gatekeeper-system \
+    --set featureFlags.RATIFY_EXPERIMENTAL_DYNAMIC_PLUGINS=true \
+    --set featureFlags.RATIFY_CERT_ROTATION=true
+```
+Refer to the Ratify [documentation](https://ratify.dev/docs/1.0/quickstarts/quickstart-manual/) if you need to customize the helm chart installation.
+
+After deploying Ratify, we will download the Venafi CodeSign Protect notation plugin to the Ratify pod using the [Dynamic Plugins feature](../reference/dynamic-plugins.md):
+
+```shell
+cat > venafi-notation-plugin.yaml << EOF
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: Verifier
+metadata:
+  name: notation-venafi-csp-plugin
+spec:
+  name: notation-venafi-csp
+  artifactTypes: application/vnd.oci.image.manifest.v1+json
+  source:
+    artifact: ghcr.io/venafi/notation-venafi-csp:linux-amd64-latest
+EOF
+
+kubectl apply -f venafi-notation-plugin.yaml
+```
+
+Next we will need to deploy the trusted Root certificate used to issue the signing identity referenced above:
+
+```shell
+cat > venafi_root.yaml << EOF
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: CertificateStore
+metadata:
+  name: ratify-notation-ztpki-cert
+  namespace: gatekeeper-system
+spec:
+  provider: inline
+  parameters:
+    value: |
+      -----BEGIN CERTIFICATE-----
+      ...
+      -----END CERTIFICATE-----
+EOF
+
+kubectl apply -f venafi_root.yaml
+```
+
+Finally, we will create a verifier that specifies the trust policy to use when verifying signatures. In this guide, we will use a trust policy that only trusts images signed by the Venafi CodeSign Protect signing identity we created earlier:
+
+```shell
+cat > notation-verifier.yaml << EOF
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: Verifier
+metadata:
+  name: verifier-notation
+spec:
+  name: notation
+  artifactTypes: application/vnd.cncf.notary.signature
+  parameters:
+    verificationCertStores:
+      certs:
+        - ratify-notation-inline-cert
```

Comment:
Hi, Should this be `ratify-notation-ztpki-cert` instead of 'ratify-notation-inline-cert'

Reply:
will be using ratify-notation-inline-cert.

Reply:
Hi, the inline cert is defined with value.[notationCert](https://github.com/deislabs/ratify/blob/main/charts/ratify/templates/inline-certificate-provider.yaml#L14l) ,could you clarify where this value is passed in to configured? thanks!

Reply:
Ohh i see, there was renamed to ratify-notation-ztpki-cert. thanks for the update.